src/sql/src/plan/statement/ddl.rs

```diff
@@ -1513,6 +1513,24 @@ pub(crate) fn load_generator_ast_to_generator(
                 count_clerk,
             }
         }
+        mz_sql_parser::ast::LoadGenerator::Ldbc => {
+            let LoadGeneratorOptionExtracted { scale_factor, .. } = options.to_vec().try_into()?;
+            let sf: i64 = match scale_factor {
+                None => 1,
+                Some(sf) => match i64::try_cast_from(sf) {
+                    Some(i) => i,
+                    None => sql_bail!("unsupported scale factor: {sf}"),
+                },
+            };
+            // See https://github.com/ldbc/ldbc_snb_bi/blob/main/snb-bi-pre-generated-data-sets.md
+            // for valid scale factors and URLs.
+            match sf {
+                1 | 3 | 10 | 30 | 100 | 300 => {}
+                _ => sql_bail!("unsupported scale factor: {sf}"),
+            }
+            let urls = vec![format!("https://pub-383410a98aef4cb686f0c7601eddd25f.r2.dev/bi-pre-audit/bi-sf{}-composite-merged-fk.tar.zst", sf)];
```

Comment:
Hardcoding this URL into a production codepath seems sketchy! Is this going to be behind a feature flag and only for development use? Or is this something end users will be able to run?

Reply:
Should be something end users can run. Do you have a recommendation? The URLs are described at https://github.com/ldbc/ldbc_snb_bi/blob/main/snb-bi-pre-generated-data-sets.md. I think asking users to specify the URL is wrong since it will only rarely change, if ever. Allowing a dynamic URL is also annoying because we can't use rust's string formatting to sub in the scale factor without some silly stuff.

Reply:
Idea: HOST and DIRECTORY can be optional WITH options in `CREATE LOAD GENERATOR`. We then make a URL as `format!("http://{HOST}/{DIR}/bi-sf{sf}-composite-merged-fk.tar.zst")` (or more probably using a URL builder instead of string parsing). This is like fine? Seems like a bunch of extra typing for a thing that probably won't ever be needed.

Reply:
I also think `HOST/DIRECTORY` would be nice to have configurable. The reason is so that a test can specify an invalid host and then make sure that Mz startup and general operation is not impacted. Given that we will allow users to run this, any issues with this source should be at most source-level errors, but not hangs or panics.

Reply:
I've made this a LD flag.